particle/particle/utilities.py

```diff
@@ -64,3 +65,74 @@ def str_with_unc(value, upper, lower=None):
         return '{value:{fsv}} + {upper:{fse}} - {lower:{fse}}'.format(value=value,
                                                                       upper=upper, lower=lower,
                                                                       fsv=fsv, fse=fse)
+
+list_name_greek_letters = (
+    'Alpha',
+    'Beta',
+    'Chi',
+    'Delta',
+    'Epsilon',
+    'Eta',
+    'Gamma',
+    'Iota',
+    'Kappa',
+    'Lambda',
+    'Mu',
+    'Nu',
+    'Omega',
+    'Omicron',
+    'Phi',
+    'Pi',
+    'Psi',
+    'Rho',
+    'Sigma',
+    'Tau',
+    'Theta',
+    'Upsilon',
+    'Xi',
+    'Zeta',
+    'alpha',
+    'beta',
+    'chi',
+    'delta',
+    'epsilon',
+    'eta',
+    'gamma',
+    'iota',
+    'kappa',
+    'lambda',
+    'mu',
+    'nu',
+    'omega',
+    'omicron',
+    'phi',
+    'pi',
+    'psi',
+    'rho',
+    'sigma',
+    'tau',
+    'theta',
+    'upsilon',
+    'xi',
+    'zeta'
```

Comment:
Can't you list only one of the two caps versions, then use titlecase or lower to make the other half?

Reply:
That won't work because unicodedata.lookup needs to know whether the greek letter name is for a capital letter or not, see the snippet
```
    if letter == letter.lower():
        return unicodedata.lookup('GREEK SMALL LETTER %s' % letter.upper())
    else:
        return unicodedata.lookup('GREEK CAPITAL LETTER %s' % letter.upper())
```
If I give a lowercase letter I would need a Boolean flag ...

Unless you just mean creating the second part of the list `list_name_greek_letters` on the fly?

Reply:
I did mean that, but this would work:

```python
return unicodedata.lookup('GREEK {case} LETTER {name}'.format(
    case = 'LOWER' if letter == letter.lower() else 'UPPER',
    name = letter.upper()))
```

PS: Please use `.format` instead of `%`.

Reply:
I will commit this nice short return statement you suggest above.